user-org-service/app/utils/validator/uservalidator/UserModelValidator.java

```diff
@@ -0,0 +1,20 @@
+package utils.validator.uservalidator;
```

Comment:
use utils.validator.service.impl.uservalidator

Reply:
sir is there any specific need for service. impl.??

Reply:
Because you define interface and it have different impl then better to use package naming.